Resources/Prototypes/Reagents/Consumable/Drink/alcohol.yml

```diff
@@ -319,6 +346,15 @@
   color: "#360000"
   spritePath: blackrussianglass.rsi
 
+- type: reagent
+  id: BlankPaper
+  name: blank paper
+  parent: BaseAlcohol
+  desc: A fizzy cocktail for those looking to start fresh. #This should in the future be able to reinstiute the oath of silence within a mime.
```

Comment:
I can code this if you want

Reply:
go ahead, I would but I have no knowledge on C# atm lmao